task/git-clone-oci-ta/0.1/git-clone-oci-ta.yaml

```diff
@@ -237,6 +243,7 @@ spec:
           exit "${EXIT_CODE}"
         fi
         printf "%s" "${RESULT_SHA}" >"$(results.commit.path)"
+        printf "%s" "${RESULT_SHA::$(params.shortCommitLength)}" > "$(results.short-commit.path)"
```

Comment:
It would be better to use `git rev-parse --short=<length>` in the unlikely case where a prefix of the configured length is not unique within the repo

Reply:
Sounds good. I went back and forth on using `rev-parse` or expansion. That's a good argument for the former.